### PR TITLE
Add metaSPAdes wrapper.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: tools/metaspades/metaspades.xml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                </param>
+                <when value="separate">
+                    <param name="fwd_reads" type="data" format="fastq" label="Forward reads" help="FASTQ format" />
+                    <param name="rev_reads" type="data" format="fastq" label="Reverse reads" help="FASTQ format" />
+                </when>
+                <when value="interleaved">
+                    <param name="interleaved_reads" type="data" format="fastq" label="Interleaved paired reads" help="FASTQ format" />
+                </when>
+                <when value="paired-collection">
+                    <param name="fastq_collection" type="data_collection" label="Paired-end reads collection" optional="false" format="fastq" collection_type="paired" help="FASTQ format" />
```
Comment (bgruening) [inline tools/metaspades/metaspades.xml]: optional="false" is redundant ... isn't it?
@davebx why you don't need to have the repeat blog in your test section? Is this a Galaxy bug?

File: tools/metaspades/metaspades.xml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    </tests>
+    <help>
+<![CDATA[
+**What it does**
+
+SPAdes – St. Petersburg genome assembler – is intended for both standard isolates and single-cell MDA bacteria assemblies. See http://bioinf.spbau.ru/en/spades for more details on SPAdes.
+
+This wrapper runs SPAdes 3.9, collects the output, and throws away all the temporary files. It also produces a tab file with contig names, length and coverage.
+
+**License**
```
Comment (bgruening) [inline tools/metaspades/metaspades.xml]: Can we move all of this into a README file. It feels wired to have so many things in the tool help.

File: tools/metaspades/metaspades.xml (hunk 1 above)
Comment (davebx) [inline tools/metaspades/metaspades.xml]: It's not required if you're only testing with one instance in the repeat.

Comment (bgruening) [inline tools/metaspades/metaspades.xml]: Interesting ... didn't know that! Thanks!

File: tools/metaspades/metaspades.xml (hunk 2 above)
Comment (davebx) [inline tools/metaspades/metaspades.xml]: That specific line doesn't really need to be there, I think. And the rest can be moved to a README.md file, you're right.

**Comment by bgruening** [comment]
@davebx thanks a lot and happy new year!

**Pull request merged**